addon/components/base/bs-form/element/control/input.js

```diff
@@ -32,7 +32,8 @@ export default Control.extend({
     'step',
     'form',
     'spellcheck',
-    'title'
+    'title',
+    'ariaDescribedby:aria-describedBy'
```

Comment:
Okay. Should we refactor this to a mixin so it's in one place and can optionally be pulled into controls?